exo/tinychat/index.html

```diff
@@ -29,8 +29,8 @@
     <div x-show="errorMessage" x-transition.opacity x-text="errorMessage" class="toast">
     </div>
 <div class="model-selector">
-<select @change="if (cstate) cstate.selectedModel = $event.target.value" x-model="cstate.selectedModel">
-<option selected="" value="llama-3.2-1b">Llama 3.2 1B</option>
+<select>
+<option value="llama-3.2-1b">Llama 3.2 1B</option>
```

Comment:
shouldn't this option have a `selected` property so that it's by default selected

Reply:
the first item in the list is selected by default in <select> tags